sources/functions/rtorrent

```diff
@@ -202,11 +202,24 @@ function build_rtorrent() {
     patch -p1 < /etc/swizzin/sources/patches/rtorrent/lockfile-fix.patch >> "$log" 2>&1
     #apply xmlrpc-fix to all rtorrents
     patch -p1 < /etc/swizzin/sources/patches/rtorrent/xmlrpc-fix.patch >> "$log" 2>&1
-    #use pkgconfig for cppunit if 0.9.6
     stdc=
-    if [[ ${rtorrentver} == "0.9.6" ]]; then
+    if [[ ${rtorrentver} == "0.9.8" ]]; then
```

Comment:
This would be the perfect place for a case statement -- things are getting a bit more verbose and the case would help clean up readability.